app/frontend/wallet/helpers/cryptoProviderUtils.ts

```diff
@@ -25,8 +25,12 @@ export function getWalletName(providerType: CryptoProviderType): WalletName {
 }
 
 export function isHwWallet(
-  providerType: CryptoProviderType
+  providerType: CryptoProviderType | undefined | null
 ): providerType is HwWalletCryptoProviderType {
+  if (providerType == null) {
+    return false
+  }
```

Comment:
If it's undefined, it will bubble down to `.includes` and always return false, wouldn't it be better to have 
```suggestion
  if (providerType == null || providerType == undefined) {
    return false
  }
```
so that it's more transparent and caught earliest possible?

Reply:
I personally see `== null` check as an idiomatic way of expressing "equal to null or undefined" so I would soo the `== undefined` check as redundant and may be actually a bit distracting as one may legitimately wonder why there is such otherwise redundant check

It's even regarded as the standard way to check for null and undefined simultaneously: https://stackoverflow.com/a/21273362/2657249

Reply:
Ohhhh I didn't notice that it's `==`, sorry. I'm too used to being very explicit :D